.circleci/config.yml

```
@@ -478,12 +478,13 @@ jobs:
           name: Deploy to PyPI
           command: |
             # Ask github actions to deploy for us
+            export PAYLOAD={"ref":"${CIRCLE_TAG}","inputs":{}}
             curl \
               -X POST \
               -H "Accept: application/vnd.github.v3+json" \
-              -H "Authorization: token ${GITHUB_TOKEN}" \
-              https://api.github.com/repos/${CIRCLE_PROJECT_USERNAME}/${CIRCLE_PROJECT_REPONAME}/dispatches \
-              -d '{"event_type":"deploy-release","client_payload":{}}'
+              -H "Authorization: Bearer ${GITHUB_TOKEN}" \
+              https://api.github.com/repos/${CIRCLE_PROJECT_USERNAME}/${CIRCLE_PROJECT_REPONAME}/dispatches/deploy-release \
```

It seems like the [order is different](https://docs.github.com/en/rest/actions/workflows#create-a-workflow-dispatch-event)

```
/repos/{owner}/{repo}/actions/workflows/{workflow_id}/dispatches
```

```suggestion
https://api.github.com/repos/${CIRCLE_PROJECT_USERNAME}/${CIRCLE_PROJECT_REPONAME}/deploy-release/dispatches \
```

---

Can create an API token and test this out? I tried a bunch of different variations like this and none of them were working for me. But maybe I was just being dumb.

---

I removed the `type: [deploy-release]` part in [deploy_release.yml](https://github.com/pyodide/pyodide/blob/b0dc19e4a169c9dcf97e21fe0ac2c635a0900f13/.github/workflows/deploy_release.yml#L4), and I was able to run a workflow with curl.

I think it was blocking a workflow run (it is [mentioned in `repository_dispatch` docs](https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#repository_dispatch), but probably it was also affecting `workflow_dispatch`:

```
When you make a request to create a repository_dispatch event, you must specify an event_type to describe the activity type.
By default, all repository_dispatch activity types trigger a workflow to run.
You can use the types keyword to limit your workflow to run when a specific event_type value is sent in the repository_dispatch webhook payload.
```